doc/netplan.md

```diff
@@ -242,8 +242,33 @@ Virtual devices
     configured and DHCP is disabled, the interface may still be brought online,
     but will not be addressable from the network.
 
+    In addition to the addresses themselves one can specify configuration
+    parameters as mappings. Current supported options are:
+
+    ``lifetime`` (scalar)
+    :    Default: ``forever``. This can be ``forever``, ``infinity`` or ``0`` and
+         corresponds to the ``PreferredLifetime`` option in
```

Comment:
"forever" and "infinity" are synonyms.  Why do we want to support two names for the same thing?  Let's just pick one and use it

Reply:
Supporting both keywords has almost no additional cost compared to supporting only one of them, as these are mapped directly to the networkd backend (where both of them are supported). I didn't initially see any issues in having them both in netplan as well, but maybe I'm missing something? Would there be an issue in keeping this parity between netplan's `lifetime` and networkd's `PreferredLifetime`?

Reply:
We don't automatically use the same terms that the backends do, in order to have a consistent and clean schema.  Having two names for the same thing is not particularly clean.  It may be easier to write, but it's one more thing to know when reading.

I think 'forever' is a better choice here than 'infinity'.

Reply:
Ah, that's a good point! Thanks for explaining, I agree on the clean schema.
I've removed `infinity` in favour of `forever`, thank you!